Dockerfile

```diff
@@ -191,6 +202,11 @@ RUN find $ROOTFS/etc/rc.d/ $ROOTFS/usr/local/etc/init.d/ -exec chmod +x '{}' ';'
 # Change MOTD
 RUN mv $ROOTFS/usr/local/etc/motd $ROOTFS/etc/motd
 
+# Dirty hack to make the bootscript add an environment file to the .ssh/ directory
+RUN echo "touch /home/docker/.ssh/environment" >> $ROOTFS/bootscript.sh
+RUN echo "echo \"PATH=/usr/local/sbin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin\" >> /home/docker/.ssh/environment" >> $ROOTFS/bootscript.sh
+RUN echo "touch ./executed.log" >> $ROOTFS/bootscript.sh
```

Comment:
It turns out that if we don't do this, the path that can be reached by ssh doesn't contain /usr/local/sbin for instance, thus the ip show command can't be found by boot2docker-cli, and boot2docker up fails to start the machine.